### PR TITLE
Feature/add logging

### PR DESCRIPTION
**Pull request by rosealti**

### CODE REVIEW

**Comment by TheNorthEestern** [comment]
What has changed here? Are there going to be more changes to come soon?

File: python/edgar_filing_searcher/parsers/parser_class.py [hunk 1]
```diff
@@ -86,15 +88,24 @@ def _parse_primary_doc_accepted_filing_date(primary_doc_root):
...
+        logging.debug('Infotable XML url parsed %s', infotable_xml_url)
 
         root = self._parse_primary_doc_root(primary_doc_xml_url)
+        logging.debug('Root parsed %s', root)
         cik = self._parse_primary_doc_cik(root)
+        logging.debug('CIK parsed %s', cik)
         company_name = self._parse_primary_doc_company_name(root)
+        logging.debug('Company name parsed %s', company_name)
         filing_date = self._parse_primary_doc_accepted_filing_date(root)
+        logging.debug('Filing date parsed %s', filing_date)
```
Comment (TheNorthEestern) [inline python/edgar_filing_searcher/parsers/parser_class.py:108]: This level of logging is really great for diagnosing issues, but I can't help but feeling that this makes the code a little harder to refactor.

Comment (rosealti) [reply python/edgar_filing_searcher/parsers/parser_class.py]: Addressed the issue

**Review by TheNorthEestern — APPROVED** [review]
LGTM

**Pull request merged**